cmd/troubleshoot/cli/analyze.go

```diff
@@ -27,7 +29,17 @@ func Analyze() *cobra.Command {
 
 			logger.SetQuiet(v.GetBool("quiet"))
 
-			result, err := analyzer.DownloadAndAnalyze(v.GetString("url"), "")
+			filename := v.GetString("spec")
+			var analyzersSpec string
+			if len(filename) > 0 {
+				out, err := ioutil.ReadFile(filename)
```

Comment:
would it be possible to use the same "read file if local path, get URL if url" logic here as in https://github.com/replicatedhq/troubleshoot/blob/v0.9.25/pkg/analyze/download.go#L73 ?

Reply:
The support-bundle URL doesn't make sense (see my comment above about the workflow).  But your right, accepting an analyzer URL does make sense here.  I'll add it.